PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/function/FunctionAppConfigFormPanelAdvance.java

```diff
@@ -113,11 +114,16 @@ private void createUIComponents() {
                                                   .build();
 
         appServiceMonitorPanel = new AppServiceMonitorPanel(project);
-        appServiceMonitorPanel.setApplicationLogVisible(true);
+        appServiceMonitorPanel.setWebServerLogVisible(false);
         appServiceMonitorPanel.setData(MonitorConfig.builder().applicationInsightsConfig(insightsConfig).build());
 
-        appServiceConfigPanelAdvanced.getSelectorSubscription().addItemListener(event -> {
+        appServiceConfigPanelAdvanced.getSelectorSubscription().addActionListener(event -> {
```

Comment:
what is the difference from `addItemListener`?

Reply:
just want to trigger update app settings table when selected item changed, no need for `ItemListener` which may be triggered both select and deselect